src/test/java/uk/gov/hmcts/reform/sscs/callback/handlers/SendToBulkPrintHandlerTest.java

```diff
@@ -98,6 +98,21 @@ public void givenANonBulkPrintEvent_thenReturnFalse() {
         assertFalse(handler.canHandle(SUBMITTED, callback));
     }
 
+    @Test
+    @Parameters({"SEND_TO_DWP", "VALID_APPEAL", "INTERLOC_VALID_APPEAL", "RESEND_TO_DWP"})
```

Comment:
Nice to have: you can merge these two tests into one and reuse the same code for all these test scenarios. You just need to pass one more parameter which is the expectation true or false.